### PR TITLE
v0.6.0 release: STU1 for balloting

### PR DESCRIPTION
**Pull request by nkarip**
These are the changes we plan to submit to HL7 for our STU1 (Standard for Trial Use) ballot.

This will be submitted to our sponsoring HL7 work group (Public Health) today (July 22). Content must be locked for ballot by August 8th. The ballot is open from August 13th through September 13th.

The ballot-ready release that comes out before August 8th (which we expect will be very similar to v0.6.0) will likely be the final "stable" release of this IG outside the HL7 process. We expect the next major release to be the official STU1 release after reconciling HL7 ballot comments sometime this fall.

If you notice any issues with v0.6.0 that need to be resolved before balloting, they **must** be resolved before August 8 so please let us know right away by [logging a GitHub issue](https://github.com/HL7/fhir-shc-vaccination-ig/issues/new/choose).

Going forward, the best ways to stay involved with the IG development process are:

1. To join the [vaccination listserv at HL7](http://lists.hl7.org/read/?forum=vaccination) ([sign up here](https://www.hl7.org/Special/committees/pher/listserv.cfm)) for announcements about our meetings
2. Continue to monitor [smart/health-cards](https://chat.fhir.org/#narrow/stream/284830-smart.2Fhealth-cards) on Zulip
3. Attend our meetings which are typically every 2 weeks at 12pm Eastern (search for "SMART Health Card" [on this page](https://www.hl7.org/concalls/CallDirectory.aspx)) -- but will likely be on hiatus until after balloting concludes September 13th

----

# Changelog

## Breaking changes

This includes any changes that would potentially cause a validation error or warning for a resource that validated successfully against the previous version of the IG.

- General clean-up of DM profiles to properly reflect our general conformance criteria that generally only MS elements should be populated. This is technically a breaking change, but should not affect implementers who were already following these conformance criteria.
- Made `Bundle.entry[slice].resource` required, which reflects the existing prose guidance. This is technically a breaking change, but given that any SHCs missing `resource` would be fundamentally broken already, it should not adversely affect implementers.
- Changed the vaccine-related SNOMED value set to only include descendents of `787859002` but not `787859002`. Previously, `787859002` ("vaccine product") was included in the value set, but this is not a meaningful code from this IG's perspective. This is technically a breaking change because the value set binding is required, but should not affect implementers.

## Non-breaking major changes
- Restructuring IG to present the DM profiles as the "primary" option, and the AD profiles as a "fallback". The default tab changed from diff to snapshot to support this.
- Introduced the concept of "profile groups", which include one or more pairs of DM/AD profiles. For example, all the lab test result profiles of Observation are in a single "profile group." There is a new profile group-specific navigation section at the top of each page related to that group. There is also a new [`profiles.html`](http://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/branches/dev/profiles.html) page that lists all the profiles in their groups (replacing the drop-down menu, which was getting too cumbersome with the number of profiles we have).
- Clarified the conformance guidance around identifying specific vaccinations in [`vaccination.html`](http://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/branches/dev/vaccination.html).
- Changed IG identifier to `hl7.fhir.uv.shc-vaccination` (approved by HL7) in preparation for balloting.
- Renamed profiles and value sets to have a consistent naming scheme. Names and URIs now begin with `shc`. This does not affect implementations (as conforming resources don't include profile or value set names), but _may_ affect tooling if profile names are hard-coded.
- Added a new patient profile to differentiate between "General" and "US-specific". The only difference is the US-specific profile does not allow for `identifier` to be set while the General profile does. This is a relaxing of a constraint so it's a non-breaking change. See [`patient.html`](http://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/branches/dev/patient.html) for guidance for implementers.

## Minor changes
- Removed [`conformance.html`](http://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/branches/dev/conformance.html) and moved the content to the new [`profiles.html`](http://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/branches/dev/profiles.html) page.
- Bug fixes to invariants
- Allow `referenceRange` for `valueString` lab results
- Add ATC slice for our Immunization profile, which was inadvertently left out of the last release
- Removed experimental vaccination reaction profile -- this was a profile of Observation, which we no longer believe is the correct resource to profile for this.
- Updated `ignoreWarnings.txt` to get a clean QA report. Reasons for ignored warnings are commented in that file. 
- Added [`contact.html`](http://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/branches/dev/contact.html) page with the contact information for the IG authors.

### CODE REVIEW

**Review by masnick — APPROVED** [review]

File: jira-spec-artifacts.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Content for https://github.com/HL7/JIRA-Spec-Artifacts/blob/master/xml/FHIR-uv-shc-vaccination.xml
+# Run script/artifacts-for-jira-spec to generate XML to STDOUT
+specification:
+  gitUrl: "https://github.com/HL7/fhir-shc-vaccinationm-ig"
+  url: "http://hl7.org/fhir/fhir-shc-vaccinationm-ig"
```
Comment (p9g) [inline jira-spec-artifacts.yaml:5]: typo

Comment (p9g) [reply jira-spec-artifacts.yaml:5]: do we want -ig in "url"?

File: jira-spec-artifacts.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  ciUrl: "http://build.fhir.org/ig/HL7/fhir-shc-vaccinationm-ig"
```
Comment (p9g) [inline jira-spec-artifacts.yaml:6]: same typo here

File: jira-spec-artifacts.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  ballotUrl: "http://hl7.org/fhir/uv/fhir-shc-vaccination/2021Sep"
+  defaultWorkgroup: "pher"
+  defaultVersion: "0.6.0"
+
+versions:
+  - current: http://build.fhir.org/ig/HL7/fhir-shc-vaccinationm-ig
```
Comment (p9g) [inline jira-spec-artifacts.yaml:12]: same typo here

File: jira-spec-artifacts.yaml (hunk 1 above)
Comment (masnick) [reply jira-spec-artifacts.yaml]: > do we want -ig in "url"?

https://github.com/HL7/fhir-mCODE-ig has `-ig` so I think that's the convention?

**Pull request merged**